wgengine/magicsock/magicsock.go

```diff
@@ -4192,4 +4216,10 @@ var (
 	// metricDERPHomeChange is how many times our DERP home region DI has
 	// changed from non-zero to a different non-zero.
 	metricDERPHomeChange = clientmetric.NewCounter("derp_home_change")
+
+	// Disco packets received through AF_PACKET read path
+	metricRecvDiscoPacketIPv4        = clientmetric.NewCounter("magicsock_disco_recv_packet_ipv4")
```

Comment:
these counter metric names don't really say that they're AF_PACKET/bpf'ed

Reply:
Updated to explicitly say "bpf". Also removed the "invalid" counters, since we already track bad disco packets in more detail in the main handler.